cli.js

```diff
@@ -72,6 +73,9 @@ codemods.sort(utils.sortByVersion);
 
 var versions = utils.getVersions(codemods);
 
+var avaConf = pkgConf.sync('ava');
```

Comment:
> Also, need to handle possibility ava section is not defined.

That's handled well. If there's no package, if the `ava` section is empty/non-existent, then it all evaluates to `{}`, and files will then take the `defaultFiles` value.